boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832.dts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2017 Shawn Nock <shawn@monadnock.ca>
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2021 morsisko
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832_qfaa.dtsi>
+
+/ {
+	model = "EBYTE E73-TBB NRF52832";
+	compatible = "ebyte,e73-tbb-nrf52832";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+			label = "Red LED1";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+			label = "Red LED2";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 17>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 14 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button SW1";
+		};
+		button1: button_1 {
+			gpios = <&gpio0 13 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button SW2";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		pwm-led0 = &pwm_led0;
+		sw0 = &button0;
+		sw1 = &button1;
+		bootloader-led0 = &led0;
+	};
+};
+
+&adc {
+	status = "okay";
```

Comment:
As far as I remember I enabled this because otherwise the test `tests/drivers/adc/adc_api/src/test_adc.c` fails. "adc" instance is also enabled by default in the https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/nrf52dk_nrf52832/nrf52dk_nrf52832.dts#L130 so I thought this is ok.

Reply:
I think we've had this discussion many times, we should finally settle on something and move on. To make things maintainable upstream, I think it's ok to have devices as "okay" and defer enablement to, e.g., `CONFIG_ADC=y` in this case. If we don't do that, then we are forced to have per-board overlays in each sample requiring ADC.

https://github.com/zephyrproject-rtos/zephyr/issues/42998

Reply:
Should I left it as is?

Reply:
I'm OK with that for now.